### PR TITLE
redo pull request #58 on Composite device 

### PR DESCRIPTION
**Pull request by mcr**
This changes may of the Composite Attester->Composite Device.
It updates the diagram a bit, but it does not go as far as #58. The terms "lead Attester" are retained, but not as new term ("lead", not "Lead").  The Attesting Environment is not pluralized.

### CODE REVIEW

File: draft-ietf-rats-architecture.md [hunk 1]
```diff
@@ -419,19 +419,20 @@ environment (after the initial one that is a root-of-trust) has the duty
...
 Each sub-entity has at least one Attesting Environment collecting the claims
 from at least one Target Environment, then this sub-entity generates Evidence
 about its trustworthiness. Therefore each sub-entity can be called an Attester.
-Among these Attesters, there may be only some, which can be called Lead Attesters,
-that have the ability to communicate with the Verifier. Other Attesters
-don't have this ability, but they are connected to the Lead Attesters via internal
-links or network connections, and they are appraised by the Lead Attester's.
+Among all the Attesters, there may be only some which have the ability to communicate
+with the Verifier while others do not.
+Those that do not are connected to the lead Attesters via internal
```
Comment (dthaler) [inline draft-ietf-rats-architecture.md]: "lead Attester" should be singular, delete the s

File: draft-ietf-rats-architecture.md [hunk 2]
```diff
@@ -419,19 +419,20 @@ environment (after the initial one that is a root-of-trust) has the duty
...
+links (or private network connections), and they are appraised by the lead Attester.
```
Comment (dthaler) [inline draft-ietf-rats-architecture.md]: "and they are appraised by the lead Attester" is an anachronism since the Attester role is not one that does appraisal, only Verifiers and Relying Parties do appraisal.   I think we either need to omit this phrase, or reword it.  (Omitting is more expedient in the short term.)

File: draft-ietf-rats-architecture.md [hunk 3]
```diff
@@ -419,19 +419,20 @@ environment (after the initial one that is a root-of-trust) has the duty
...
 
-For example, a carrier-grade router is a composite device consisting of a chassis and multiple slots.
+For example, a carrier-grade router is consisted of a chassis and multiple slots.
```
Comment (dthaler) [inline draft-ietf-rats-architecture.md]: grammar: s/is consisted of/consists of/ 
(or revert the change if you prefer the original)

File: draft-ietf-rats-architecture.md [hunk 4]
```diff
@@ -440,15 +441,15 @@ while other slots cannot. But other slots can communicate with the main
 slot by the links between them inside the router. So the main slot collects
 the Evidence of other slots, produces the final Evidence of the whole router and
 conveys the final Evidence to the Verifier. Therefore the router is a Composite
-Attester, each slot is an Attester, and the main slot is the Lead Attester.
+Device, each slot is an Attester, and the main slot is the transiting Attester.
```
Comment (dthaler) [inline draft-ietf-rats-architecture.md]: s/transiting/relaying/

File: draft-ietf-rats-architecture.md [hunk 5]
```diff
@@ -460,26 +461,22 @@ each router is an Attester and the main router is the Lead Attester.
...
-appraises them using Endorsements and Appraisal Policies (obtained the
-same way as any other Verifier).
-The Lead Attesting Environment combines the Attestation Results into
-the final Evidence of the whole Composite Attester which is sent off to the remote
-Verifier, which may treat the Claims obtained from the local Attestation Results
-as if they were Evidence. In this situation, the trust model described
-in {{trustmodel}} is also suitable for the local Verifier.
+An entity can take on multiple RATS roles (e.g., Attester, Verifier, Relying
+Party, etc.) at the same time. The combination of roles can be arbitrary.
+For example, in this Composite Device scenario, the entity inside of
```
Comment (dthaler) [inline draft-ietf-rats-architecture.md]: grammar: s/inside of/inside/

File: draft-ietf-rats-architecture.md [hunk 6]
```diff
@@ -460,26 +461,22 @@ each router is an Attester and the main router is the Lead Attester.
...
+the lead Attester can also take on the role of a Verifier, and the
+outside entity of Verifier takes on the role of a Relying Party.
```
Comment (dthaler) [inline draft-ietf-rats-architecture.md]: grammar: s/takes/can take/

File: draft-ietf-rats-architecture.md [hunk 7]
```diff
@@ -460,26 +461,22 @@ each router is an Attester and the main router is the Lead Attester.
...
+After collecting the Evidence of other Attesters, this inside Verifier
+verifies them using Endorsements and Appraisal Policies (obtained the
+same way as any other Verifier), to generate Attestation Results.
+The inside Verifier then sends the Attestation Results of other Attesters,
+whether in the same conveyance protocol as the Evidence or not,
+to the outside Relying Party.
```
Comment (dthaler) [inline draft-ietf-rats-architecture.md]: should this say "to the outside Verifier"?

File: draft-ietf-rats-architecture.md (hunk 1 above)
Comment (mcr) [reply draft-ietf-rats-architecture.md]: fixed.
